data_augmentation.py

```diff
@@ -0,0 +1,28 @@
+from keras.preprocessing.image import ImageDataGenerator
+
+shift = 0.1
+
+
+def create_datagen():
+    """
+    Function creates ImageDataGenerator used in process of augmentation Smile-Warrior dataset to increase accuracy of
```

Comment:
I do not think comment is useful here. If user wants to know the details, he will check the docs for ImageDataGenerator.